googletest/include/gtest/internal/gtest-port.h

```diff
@@ -616,12 +618,21 @@ struct _RTL_CRITICAL_SECTION;
 // Determines if hash_map/hash_set are available.
 // Only used for testing against those containers.
 #if !defined(GTEST_HAS_HASH_MAP_)
-# if _MSC_VER
+# if defined(_MSC_VER) && (_MSC_VER < 1900)
 #  define GTEST_HAS_HASH_MAP_ 1  // Indicates that hash_map is available.
 #  define GTEST_HAS_HASH_SET_ 1  // Indicates that hash_set is available.
 # endif  // _MSC_VER
 #endif  // !defined(GTEST_HAS_HASH_MAP_)
 
+// Determines if unordered_map/unordered_set are available.
+// Only used for testing against those containers.
+#if !defined(GTEST_HAS_UNORDERED_MAP_)
+# if defined(_MSC_VER) && (_MSC_VER >= 1900)
```

Comment:
std::unordered_set and std::unordered_map tests should also be enabled on other platforms, I would say this check should be #if GTEST_LANG_CXX11